sql/core/src/main/scala/org/apache/spark/sql/execution/BaseScriptTransformationExec.scala

```diff
@@ -72,6 +72,10 @@ trait BaseScriptTransformationExec extends UnaryExecNode {
   protected def initProc: (OutputStream, Process, InputStream, CircularBuffer) = {
     val cmd = List("/bin/bash", "-c", script)
     val builder = new ProcessBuilder(cmd.asJava)
+      .directory(new File(SparkFiles.getRootDirectory()))
+    val path = SparkFiles.getRootDirectory() + File.pathSeparator +
```

Comment:
Shall we place `PATH` first so it won't mistakenly have the higher priority over the exectuables in `PATH`?

Reply:
> Shall we place `PATH` first so it won't mistakenly have the higher priority over the exectuables in `PATH`?

Done.